### PR TITLE
Edit player motion

### PR DESCRIPTION
**Pull request by andyytang**

### CODE REVIEW

File: survival.js [hunk 1]
```diff
@@ -87,12 +91,8 @@ var referencepoint = function(x, y) {
             ellipse((this.x - 20 - Player.x)/16 + Player.x - 370, (this.y - 20 - Player.y)/16 + Player.y + 80, 2, 2);
         }
     };
-    this.harvest = function()  {
```
Comment (chessgoose) [inline survival.js]: keep this stuff

File: survival.js [hunk 2]
```diff
@@ -138,57 +138,33 @@ var player = function(x, y) {
...
+        this.x += cos(this.dir)*this.speed;
+        this.y += sin(this.dir)*this.speed;
     };
     this.stats = function() {
         fill(0);
         text("Location: (" + round(this.x*100)/100 + ", " + round(this.y*100)/100 + ")", 20, 20);
         text("Facing: " + round((((-1*Math.sign(this.dir)+1)/2)*360 + this.dir)*100)/100 + " degrees from East", 20, 35);
     };
-    this.collectTree = function(refpoint){
-        return abs(this.x - refpoint.x) < (refpoint.w*refpoint.leaves) && abs(this.y - refpoint.y) < (refpoint.w*refpoint.leaves);
```
Comment (chessgoose) [inline survival.js]: keep this

File: survival.js [hunk 3]
```diff
@@ -138,57 +138,33 @@ var player = function(x, y) {
...
-for(var b = 0; b < 1000; b++) {
+for(var i = 0; i < 1000; i++) {
     refpoints.add(random(0, 10000), random(0, 10000));  
 }
 
-mouseClicked = function() {
-    for (var i = 0; i < refpoints.length; i++) {
-        if (Player.collectTree(refpoints[i]) === true) {
-            refpoints[i].harvest();
-        }
```
Comment (chessgoose) [inline survival.js]: Keep the mouseClicked stuff

File: survival.js (hunk 1 above)
Comment (andyytang) [reply survival.js]: ok

**Pull request merged**